centreon/src/App/ResourceConfiguration/Infrastructure/ApiPlatform/State/CreateServiceCategoryProcessor.php

```
@@ -0,0 +1,72 @@
+    /**
+     * @param TransformerInterface<ServiceCategory, ServiceCategoryResource> $transformer
+     */
+    public function __construct(
+        private CommandBus $commandBus,
+        #[Autowire(service: ServiceCategoryTransformer::class)]
+        private TransformerInterface $transformer,
+        private LegacySecurity $legacySecurity,
+    ) {
+    }
+
+    public function process($data, Operation $operation, array $uriVariables = [], array $context = []): ServiceCategoryResource
```

PHPMD (2.15.0)

Avoid unused parameters such as '$operation'. (Unused Code Rules)

(UnusedFormalParameter)